src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/link-otel-applications-kubernetes.mdx

```diff
@@ -84,201 +84,153 @@ spec:
 
-  <Step>
-    ### Configure the resource detection processor [#resource-detection]
+      ```yaml
+      processors:
+      k8sattributes:
+          auth_type: "serviceAccount"
+          passthrough: false
+          filter:
+          node_from_env_var: KUBE_NODE_NAME
+          extract:
+          metadata:
+              - k8s.pod.name
+              - k8s.pod.uid
+              - k8s.deployment.name
+              - k8s.cluster.name
+              - k8s.namespace.name
+              - k8s.node.name
+              - k8s.pod.start_time
+          pod_association:
+          - from: resource_attribute
+              name: k8s.pod.uid
+      ```
+    </Collapser>
+      <Collapser
+          id="rbac"
+          title="Configure the Kubernetes Attributes processor (RBAC)"
+      >
 
-    The `resourcedetection` processor gets host-specific information to add additional context to the telemetry data being processed through the collector. In this example, we use Google Kubernetes Engine (GKE) and Google Compute Engine (GCE) to get Google Cloud-specific metadata, including:
+      You need to add configurations for role-based access control (RBAC). The `k8sattributes` processor needs `get`, `watch`, and `list` permissions for pods and namespaces resources included in the configured filters. This [example](https://github.com/newrelic-forks/microservices-demo/blob/main/otel-kubernetes-manifests/otel-collector-agent.yaml#L43-L69) shows how to configure role-based access control (RBAC) for `ClusterRole` to give a `ServiceAccount` the necessary permissions for all pods and namespaces in the cluster.
+      </Collapser>
+      <Collapser
+          id="discovery-filter"
+          title="Configure the Kubernetes Attributes processor (discovery filter)"
+      >
 
-    * `cloud.provider` ("gcp")
-    * `cloud.platform` ("`gcp_compute_engine`")
-    * `cloud.account.id`
-    * `cloud.region`
-    * `cloud.availability_zone`
-    * `host.id`
-    * `host.image.id`
-    * `host.type`
+      When running the collector as an agent, you should apply a discovery filter so that the processor only discovers pods from the same host that it is running on. If you don't use a filter, resource usage can be unnecessarily high, especially on very large clusters. Once the filter is applied, each processor will only query the Kubernetes API for pods running on its own node.
+
+      To set the filter, use the downward API to inject the node name as an environment variable in the pod `env` section of the OpenTelemetry Collector agent configuration YAML file. To see an example, check out [GitHub](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/examples/kubernetes/otel-collector-config.yml). This will inject a new environment variable to the OpenTelemetry collector agent's container. The value will be the name of the node the pod was scheduled to run on.
```

Comment:
Instead of "check out Github" replace github with the actual name of the example

Reply:
Thank you! much better!